.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+cache:
+  directories:
+    - node_modules
+    - $HOME/.cache
+    - $HOME/.yarn-cache
+notifications:
+  email: false
+node_js:
+  - '6'
+  - '7'
```

Comment:
I would remove node 7 at the moment.
I'm quite sure the stack do not support it very well.
I would like to wait for the complete integration of all our stack before adding node 7 support